schema.sql

```diff
@@ -10,3 +10,28 @@ CREATE TABLE animals(
     );
     
     ALTER TABLE animals ADD species VARCHAR(50);
+
+
+    /* create table for owners' data.*/
+CREATE TABLE owners( 
+    id                  INT GENERATED BY DEFAULT AS IDENTITY (START WITH 1 INCREMENT BY 1), 
+    full_name            VARCHAR(50), 
+    age                  INT,
+    );
+
+    /* create table for species' data.*/
+CREATE TABLE species( 
```

Comment:
The same applies here 🤓 Kindly add a `PRIMARY KEY (id)` to species table so that we make the id row unique 📝